azerothcore/azerothcore-wotlk
# fix(Core/Spell): Implement SPELL_ATTR6_NO_PUSHBACK

Only 2 spells seem to have that flag (none of which is a cast/channel)

| ID | SpellName0 | 
| ---: | --- | 
| 54368 | Disease Cloud | 
| 67892 | Touch of the Val'kyr | 

## Changes Proposed:

This PR proposes changes to:

## Issues Addressed:

- Closes https://github.com/chromiecraft/chromiecraft/issues/5921

## SOURCE:

The changes have been validated through:

## Tests Performed:

This PR has been:


## How to Test the Changes:



.go c id 17257
.modify hp 50000000 (make sure godmod is OFF)
click the cube
you shouldn't get any pushback on the channel

## Known Issues and TODO List:




## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
